.gitmodules

```diff
@@ -1,16 +1,16 @@
-[submodule "Lombiq.Arithmetics"]
-	path = Lombiq.Arithmetics
-	url = https://github.com/Lombiq/Arithmetics.git
-	branch = dev
-[submodule "Hastlayer-Hardware-Framework---Vitis"]
-	path = Hastlayer-Hardware-Framework---Vitis
-	url = https://github.com/Lombiq/Hastlayer-Hardware-Framework---Vitis
-	branch = dev
 [submodule "tools/Lombiq.Analyzers"]
 	path = tools/Lombiq.Analyzers
 	url = https://github.com/Lombiq/.NET-Analyzers.git
 	branch = dev
-[submodule "Libraries/Lombiq.HelpfulLibraries"]
-	path = Libraries/Lombiq.HelpfulLibraries
+[submodule "src/Hardware Frameworks/Vitis"]
+	path = src/Hardware Frameworks/Vitis
```

Comment:
What's the _HardwareFramework_ folder there?

Reply:
HardwareFramework is the location in the build directory, since there it actually matters. But they don't mean the same thing, so I don't see a reason to make the names more similar.